Sources/PitchSpeller/Data Structures/DirectedEdge.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  DirectedEdge.swift
+//  PitchSpeller
+//
+//  Created by Benjamin Wetherfield on 7/1/18.
+//
+
+public struct DirectedEdge<Node, Num>: EdgeLike {
```

Comment:
I'd prefer no abbrv. in type names (`Num`). Would `Value` be too general?

Would it be helpful to nest this in `Graph`, or do you imagine it would work better outside of that?

Reply:
Definitely! That may solve a lot of problems I think in fact